apps/drupal/particle.libraries.yml

```diff
@@ -1,22 +1,24 @@
-# @TODO: THIS IS NEEDS LOTS OF WORK, THESE PATHS MAY BE NO LONGER CORRECT
+# Define Particle theme's library CSS and JS assets along with dependencies
 core:
   css:
     theme:
-      ../../dist/assets/app-drupal.styles.css:
+      /themes/particle/dist/assets/app-drupal.styles.css:
         preprocess: true
   js:
-    ../../dist/assets/commons.js:
-      preprocess: true
-    ../../dist/assets/app-drupal.js:
+    /themes/particle/dist/assets/app-drupal.js:
       preprocess: true
   # see all in Drupal's `core/core.libraries.yml`
   dependencies:
-    # - patternlab/j-dash
     - core/drupal
-# @TODO: GET THESE PULLED OUT VIA WEBPACK
-#jquery:
-#  js:
-#    ../node_modules/jquery/dist/jquery.min.js: { minified: true, weight: -20 }
-#lodash:
-#  js:
-#    ../node_modules/lodash/lodash.min.js: { minified: true, weight: -20 }
+    - core/drupalSettings
+
+# Create a custom jQuery and Lodash libraries
+# that will override the Drupal core jQuery and Underscore
+# See `libraries-override` in particle.info.yml
+particle-jquery:
```

Comment:
A suggestion, now that we have this rock solid, using `particle/particle-jquery|lodash` in `particle.info.yml` now feels weird. Let's get the library updated so we can just do `particle/jquery` and `particle/lodash` now.

Basically, just update the libraries key, no need to change the webpack output filenames ;)